src/dmd/root/response.d

```diff
@@ -14,200 +14,341 @@
 
 module dmd.root.response;
 
-import core.stdc.stdio;
-import core.stdc.stdlib;
-import core.stdc.string;
 import dmd.root.file;
 import dmd.root.filename;
 
+alias responseExpand = responseExpandFrom!lookupInEnvironment;
+
 /*********************************
- * #include <stdlib.h>
- * int response_expand(int *pargc,char ***pargv);
- *
  * Expand any response files in command line.
  * Response files are arguments that look like:
  *   @NAME
- * The name is first searched for in the environment. If it is not
- * there, it is searched for as a file name.
```

Comment:
I think the lookup order information matters. Can you rephrase it in a way that refers to `lookup` rather than the environment ?